src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -534,6 +534,24 @@ class EventView {
     return decodeColumnOrder(this.fields);
   }
 
+  getDays(): number {
+    const statsPeriod = decodeScalar(this.statsPeriod);
+    const start = decodeScalar(this.start);
+    const end = decodeScalar(this.end);
```

Comment:
Do you need to decodeScalar here? These properties should already be `string | undefined`